### DISCUSSION TITLE
osu!lazer appimage crashing

### DISCUSSION THREAD

**Discussion by chinh4** [ppy/osu#21133] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

I'm running this on arch linux and currently using the Appimage due to the provided aur package being outdated. Every time I try to start osu!lazer it shows a black screen (sometimes not at all) and returns `Bus error (core dumped)`.

### Screenshots or videos

https://user-images.githubusercontent.com/34662114/200095476-0931388f-b814-4423-8c9c-fe9b8ba3b8e3.mp4

### Version

2022.1101.0 

### Logs

[database.log](https://github.com/ppy/osu/files/9942702/database.log)
[input.log](https://github.com/ppy/osu/files/9942703/input.log)
[network.log](https://github.com/ppy/osu/files/9942704/network.log)
[performance.log](https://github.com/ppy/osu/files/9942705/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9942707/runtime.log)

**Comment by bdach** [comment]
Please attach `dmesg` output after one such crash.

**Comment by chinh4** [comment, marked as answer]
a reboot fixed it 😐

**Comment by xiaoniaoawa** [comment]
same, but reboot can not fix it.
osu lazer can run in the first time, but I exit the game and when I run it again, it crash.

`xiaoniao@xiaoniao-desktop:~/下载$ ./osu.appimage `
`Bus error (core dumped)`

**Comment by ztlwk** [comment]
Same issue here. Deleting "AuthNative.so" works

**Reply by bdach** [reply]
Do not do this, as it may result in you not being able to play online in the future.

**Reply by ztlwk** [reply]
It had shown a blank screen with bgm at the first time, but I was unable to close it properly, so I terminated the program.
Maybe I broke something then.

**Reply by ztlwk** [reply]
Is there any solution other than this? Because a reboot didn't work for me.

**Reply by bdach** [reply]
Please attach game logs as described in https://github.com/ppy/osu/discussions/15188
